src/smallMultiples.js

```diff
@@ -68,7 +68,7 @@ export default function smallMultiples(id, chart) {
         rangePolygon(this);
 
         // axis tweaks
-        var units = this.current_data[0].values.raw[0][chart.config.unit_col];
+        var units = this.raw_data[0][chart.config.unit_col];
```

Comment:
we should really clean up the data paradigm in webcharts a bit ... let's make time for that after these releases go out.